app/_src/installation/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Installation options
```

Comment:
@slonka Good catch. This page was my old plan, back when I thought about adding the old install pages back in. Since I decided not to do that after talking to Charly, I'll delete this page.